### ISSUE TITLE
you have requested a non-existent service "event_dispatcher"

### ISSUE THREAD

**Issue by fritzmg** [contao/managed-edition#7]
When downloading the `contao/managed-edition` from contao.org, you currently receive the following error message when accessing `/install.php`.
```
Fatal error: Uncaught exception 'Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException' 
with message 'You have requested a non-existent service "event_dispatcher".' 
in vendor\symfony\symfony\src\Symfony\Component\DependencyInjection\ContainerBuilder.php:803 
Stack trace: 
#0 vendor\symfony\symfony\src\Symfony\Component\DependencyInjection\ContainerBuilder.php(439): Symfony\Component\DependencyInjection\ContainerBuilder->getDefinition('event_dispatche...')
#1 vendor\contao\installation-bundle\src\Controller\InstallationController.php(125): Symfony\Component\DependencyInjection\ContainerBuilder->get('event_dispatche...') 
#2 vendor\contao\installation-bundle\src\Controller\InstallationController.php(63): Contao\InstallationBundle\Controller\InstallationController->initializeApplication() 
#3 vendor\contao\installation-bundle\src\HttpKernel\InstallationKernel.php(89): Contao\InstallationBundle\Controll in vendor\symfony\symfony\src\Symfony\Component\DependencyInjection\ContainerBuilder.php on line 803
```
Tested under

* Windows 10
* XAMPP
* PHP 5.6.21 and 7.1.1

**Comment by leofeyer** [comment]
The issue should not occur when you use the `/contao/install` route.

**Comment by fritzmg** [comment]
You are right, it only occurs when using `/install.php`, my mistake.

**Comment by leofeyer** [comment]
Fixed in contao/installation-bundle@796a39221a695fd5fac977d8ff244fe9f470b433.
